pkg/client/ghcr/ghcr.go

```diff
@@ -110,35 +104,9 @@ func getTagsFromOrgPackage(client *github.Client, owner string, repo string) ([]
 		if resp.NextPage == 0 {
 			break
 		}
+
 		opts.ListOptions.Page = resp.NextPage
 	}
-	return tags, nil
-}
 
-func getTagsFromRelease(client *github.Client, owner string, repo string) ([]api.ImageTag, error) {
```

Comment:
Using releases, was a way of being significantly quicker than searching the repo due to GitHubs aggressive rate limiting... Maybe this could be an option/annotation perhaps to enable/disable or a fallback?

Reply:
Being able to say 'I want to use release versions to figure out latest' for specific repositories sounds like a good feature. We probably need to think about the best way to do that.

I would say that we could merge this PR, which fixes things for repositories that are currently broken, and then think about that in a follow up?

Reply:
I'm happy to merge if an Issue is raised to capture this? (Once I get time to re-review things 🙈 )

Reply:
Issue raised #196 to cover this, primarily to unblock this PR :)